### DISCUSSION TITLE
Is it possible to implement in renative ?

### DISCUSSION THREAD

**Discussion by akhilashok7539** [osamaqarem/react-native-image-colors#49] [Q&A] [answered]
the platforms like    "android",
            "androidtv",
            "firetv",
            "androidwear",
            "ios",
            "web",
            "macos",
            "tvos",
            "tizen",
            "tizenmobile",
            "webos",
            "windows",
            "tizenwatch",
            "kaios",
            "firefoxos",
            "firefoxtv",
            "chromecast",
            "webtv",
            "xbox"

**Comment by osamaqarem** [comment, marked as answer]
Probably. It is just that I have no personal use case for these platforms. However if a good PR adding support to a particular platform is submitted I wouldn't mind reviewing and merging it.

**Reply by akhilashok7539** [reply]
iam facing issue getcolors undefined in running the application in androidtv because the menthod doesnt work

**Reply by osamaqarem** [reply]
Only web, Android and iOS are supported as per the README page.
